holonote/annotate/display.py

```diff
@@ -236,6 +234,15 @@ class AnnotationDisplay(param.Parameterized):
 
     data = param.DataFrame(doc="Combined dataframe of annotation data", constant=True)
 
+    nearest_2d_point_threshold = param.Number(
+        default=0.1,
+        bounds=(0, None),
+        doc="""
+        Threshold for selecting an existing 2D point; anything over
```

Comment:
It should be mentioned that the point is in the distance in data coordinates between the two dimensions and that it does not consider the unit and magnitude differences between the dimensions.